src/darwin/Framework/CHIPTests/MTRCommissionableBrowserTests.m

```diff
@@ -56,8 +56,26 @@ - (instancetype)initWithExpectation:(XCTestExpectation *)expectation
     return self;
 }
 
+- (instancetype)init
+{
+    if (!(self = [super init])) {
+        return nil;
+    }
+
+    _resultsCount = 0;
```

Comment:
ivars are zero initialized, so this whole `init` method can be omitted.

Reply:
I think having the clarity is good, fwiw.